.azure-pipelines/template-test-minimal-conda-env-install.yml

```diff
@@ -0,0 +1,140 @@
+                  if [[ "$value" == "["*"]" ]]; then
+                    repos=$(echo "$value" | jq -r '. | join(" ")')
+                  else
+                    repos=$value
+                  fi
+
+                  # Create object
+                  object_string=$( jq -n \
+                                    --arg envType "$key" \
+                                    --arg filename "$filename" \
+                                    --arg repositories "$repos" \
+                                    '{ ($envType): {name: $envType, filename: $filename, repositories: $repositories }}' )
+
+                  # Merge with the main json_obj
+                  matrix=$(echo "$matrix" | jq ". + $object_string")
+
+                  # Copy matching files to the artifact folder
+                  cp "$matchingFile" "$artifactFolder/"
+                done < <(echo "$envToRepoJson" | jq -c 'to_entries[]')
+
+                json_matrix=$(echo $matrix | jq -c .)
+
+                # Save and output the final matrix
+                echo "##vso[task.setvariable variable=matrix;isOutput=true]$json_matrix"
+          - publish: 'artifact_envFiles/'
+            artifact: envFiles
+            displayName: 'Publish environment files'
+      - job: TestInstallation
+        displayName: 'Test environment installation'
+        pool:
+          vmImage: 'windows-2022'
+        container:
+          image: mirageoscienceltd.jfrog.io/mira-docker-local/windows-servercore-ltsc2022-micromamba:latest
+          endpoint: Docker-Artifactory
+        dependsOn: DiscoverFiles
+        strategy:
+          matrix: $[ dependencies.DiscoverFiles.outputs['DiscoverFiles.matrix'] ]
+        variables:
+        - group: MIRA_PYTHON
+        steps:
+          - checkout: none
+          - download: current
+            artifact: envFiles
+          - task: PowerShell@2
+            name: ConfigureSources
+            displayName: 'Configure PyPi sources'
+            inputs:
+              targetType: 'inline'
+              script: |
+                $string = "$(repositories)"
+                $array = $string.Split(" ")
+                $extra_index_url = ""
+                $isLast = ($array.Length - 1)
+                foreach ($repo in $array) {
+                  echo "Adding repository ${repo}-pypi-${{ parameters.environment }}"
+                  $extra_index_url += "https://$(JFROG_ARTIFACTORY_USER):$(JFROG_ARTIFACTORY_TOKEN)@$(JFROG_ARTIFACTORY_URL)/pypi/${repo}-pypi-${{ parameters.environment }}/simple"
+                  if ($isLast -gt 0) {
+                    $extra_index_url += " "
+                    $isLast--
+                  }
+                }
+                echo "##vso[task.setvariable variable=extra_index_url;issecret=true]$extra_index_url"
+          - task: PowerShell@2
+            inputs:
+              targetType: 'inline'
+              script: |
+                $Env:PIP_EXTRA_INDEX_URL = "$(extra_index_url)"
```

Comment:
**issue:** it will still fetch from PyPI first, because PIP_INDEX_URL is not redefined

Reply:
**issue:** replicate the installation from Analyst

- force `PIP_NO_DEPS=1`
- force conda channels to be those of Artifactory  (can possibly be addressed in another dedicated issue)

Reply:
> * force conda channels to be those of Artifactory  ~(can possibly be addressed in another dedicated issue)~

thanks !